dask/tests/test_base.py

```diff
@@ -514,3 +514,10 @@ def my_get(dsk, keys):
 
     with dask.set_options(array_optimize=None, get=my_get):
         y.compute()
+
+
+def test_rechunk_zero_dim():
+    da = pytest.importorskip('dask.array')
+
+    x = da.ones((0,10,100), chunks=(0,10,10)).rechunk((0,10,50))
```

Comment:
flake8 will probably complain at you for not putting spaces after commas here

Reply:
The following version seems content?

```bash
flake8 --version
3.4.1 (mccabe: 0.6.1, pycodestyle: 2.3.1, pyflakes: 1.5.0) CPython 2.7.12 on Linux
```

Reply:
Do you have a `~/.config/flake8` file that's suppressing that check?